### PR TITLE
[qtinterfaceframework|opencv] fix python stuff

### PR DESCRIPTION
**Pull request by Neumann-A**
patch link https://codereview.qt-project.org/c/qt/qtinterfaceframework/+/404916 (not stable for download)

### CODE REVIEW

**Comment by Neumann-A** [comment]
@cenit: 
Do you have a fix for:
```/usr/bin/ld: /mnt/vcpkg-ci/installed/x64-linux/debug/lib/libilbc.a(checks.cc.o): relocation R_X86_64_PC32 against symbol `stdout@@GLIBC_2.2.5' can not be used when making a shared object; recompile with -fPIC```

**Comment by cenit** [comment]
looks like libilbc cmakelists.txt overwrites cmake_c_flags, removing the necessary -fPIC...

**Comment by Neumann-A** [comment]
> **looks like libilbc cmakelists.txt overwrites cmake_c_flags, removing the necessary -fPIC...**

Its worse than that:
https://github.com/TimothyGu/libilbc/blob/6adb26d4a4e159cd66d4b4c5e411cd3de0ab6b5e/CMakeLists.txt#L55

should be fixed with the latest commits

**Review by JonLiu1993 — APPROVED** [review]

File: ports/libilbc/vcpkg.json [hunk 1]
```diff
@@ -1,8 +1,15 @@
 {
   "name": "libilbc",
-  "version-string": "3.0.3",
-  "port-version": 1,
+  "version": "3.0.4",
   "description": "Open source implementation of the Internet Low Bit Rate Codec (iLBC) / RFC 3951 codec from the WebRTC project.",
   "homepage": "https://github.com/TimothyGu/libilbc",
-  "supports": "!(arm & uwp)"
+  "license": "BSD-3-Clause",
+  "supports": "!(arm & uwp)",
```
[the remark this answers is not in the document]
Comment (strega-nil-ms) [reply ports/libilbc/vcpkg.json:7]: looks like maybe something to do with mixing arm and msbuild

**Review by strega-nil-ms — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**